src/walnut.ts

```diff
@@ -213,11 +220,9 @@ const hallways: Hallway[] = [
     }),
     new Stairs(LEFT, StairNode.D2, "2015"),
     new Room("2101", RIGHT),
-    new Room("Junior High Gym", LEFT, {
-      prefix: "the",
-      aliases: ["2402", "Junior Gymnasium"],
-    }),
+    new Fork(LEFT, ConnectionNode.C2100S_TO_2401, "the tiny hallway"),
     new Room("2103", RIGHT),
+    new Fork(LEFT, ConnectionNode.C2100S_TO_2404, "the tiny hallway"),
```

Comment:
If we can think of one, we might want to have 2 different names for the 2 hallways labeled "the tiny hallway"